.github/actions/win-sign-action/action.yml

```
@@ -0,0 +1,137 @@
+        }
+
+        $successCount = 0
+        $jobs = @()
+        foreach ($file in $files) {
+            # Throttle to max 5 concurrent jobs
+            if ($jobs.Count -ge 5) {
+                $completed = $jobs | Wait-Job -Any
+                $result = $completed | Receive-Job
+                $jobs = $jobs | Where-Object { $_.Id -ne $completed.Id }
+                $completed | Remove-Job
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)